homeassistant/components/recorder/models.py

```diff
@@ -198,6 +210,33 @@ def to_native(self, validate_entity_id=True):
             return None
 
 
+class Statistics(Base):  # type: ignore
+    """Statistics."""
+
+    __table_args__ = {
+        "mysql_default_charset": "utf8mb4",
+        "mysql_collate": "utf8mb4_unicode_ci",
+    }
+    __tablename__ = TABLE_STATISTICS
+    id = Column(Integer, primary_key=True)
+    entity_id = Column(String(255))
+    period = Column(Enum(*STATISTIC_PERIODS))
+    start = Column(DATETIME_TYPE, index=True)
+    mean = Column(Float())
+    min = Column(Float())
+    max = Column(Float())
+
```

Comment:
Let's add `created_at` column.

Reply:
Because we will want to allow integrations to import historical data. In that case `start` won't match `created_at`.

Reply:
ahh, we want to be able to audit the data